src/app/users/users.component.ts

```diff
@@ -1,15 +1,22 @@
+import { HttpClient } from '@angular/common/http';
 import { Component, OnInit } from '@angular/core';
+import { IUser } from './user.interface';
 
 @Component({
   selector: 'app-users',
   templateUrl: './users.component.html',
-  styleUrls: ['./users.component.css']
+  styleUrls: ['./users.component.css'],
 })
 export class UsersComponent implements OnInit {
-
-  constructor() { }
+  constructor(private http: HttpClient) {}
```

Comment:
Never use client in a component .

Reply:
sorry dont understand  your statement. I have followed the udemy angular approach.

Reply:
Done thanks!